app/forms/instance-create.tsx

```diff
@@ -526,14 +546,28 @@ export function CreateInstanceForm() {
 const AdvancedAccordion = ({
   control,
   isSubmitting,
+  siloPools,
 }: {
   control: Control<InstanceCreateInput>
   isSubmitting: boolean
+  siloPools: Array<{ name: string; isDefault: boolean }>
 }) => {
   // we track this state manually for the sole reason that we need to be able to
   // tell, inside AccordionItem, when an accordion is opened so we can scroll its
   // contents into view
   const [openItems, setOpenItems] = useState<string[]>([])
+  const externalIps = useController({ control, name: 'externalIps' })
+  const ephemeralIp = externalIps.field.value?.find((ip) => ip.type === 'ephemeral')
+  const assignEphemeralIp = !!ephemeralIp
+  const selectedPool = ephemeralIp && 'pool' in ephemeralIp ? ephemeralIp.pool : undefined
+  const defaultPool = siloPools.find((pool) => pool.isDefault)?.name
+
+  const poolLabel = (pool: { name: string; isDefault: boolean }) => (
+    <div className="flex items-center gap-2">
+      {pool.name}
+      {pool.isDefault && <Badge>default</Badge>}
+    </div>
+  )
```

Comment:
this is a pure function, easy to either move out of render, or honestly just inline it at the call site